runelite-client/src/main/java/net/runelite/client/plugins/kourendlibrary/Book.java

```diff
@@ -59,13 +67,74 @@
 	TRISTESSAS_TRAGEDY(ItemID.TRISTESSAS_TRAGEDY, "Tristessa's Tragedy", "The Tragedy of Tristessa."),
 	TREACHERY_OF_ROYALTY(ItemID.TREACHERY_OF_ROYALTY, "The Treachery of Royalty", "The Treachery of Royalty, by Professor Answith."),
 	TRANSPORTATION_INCANTATIONS(ItemID.TRANSPORTATION_INCANTATIONS, "Transportation Incantations", "Transportation Incantations, by Amon Ducot."),
-	SOUL_JORUNEY(ItemID.SOUL_JOURNEY, "Soul Journey", "The Journey of Souls, by Aretha."),
+	SOUL_JOURNEY(ItemID.SOUL_JOURNEY, "Soul Journey", "The Journey of Souls, by Aretha."),
 	VARLAMORE_ENVOY(ItemID.VARLAMORE_ENVOY, "Varlamore Envoy", "The Envoy to Varlamore, by Deryk Paulson.");
 
-	private static final Map<Integer, Book> BY_ID = buildById();
+	/**
+	 * Map of books by id. Used in KourendLibraryPlugin when we need to get a book by its id.
+	 */
+	private static final Map<Integer, Book> BY_ID;
+
+	/**
+	 * Map of books by name. Used in KourendLibraryPlugin  when we need to get a book by its name.
+	 */
+	private static final Map<String, Book> BY_NAME;
+
+	/**
+	 * Item id of the book.
+	 */
+	@Getter
+	private final int item;
 
-	private static final Map<String, Book> BY_NAME = buildByName();
+	/**
+	 * Full name of the book used by the Library customers when requesting a book.
+	 */
+	@Getter
+	private final String name;
 
+	/**
+	 * Short name of the book used in the overlays.
+	 */
+	@Getter
+	private final String shortName;
+
+	/**
+	 * Icon of the book used in the overlays.
+	 */
+	@Getter
+	private AsyncBufferedImage icon;
+
+	/**
+	 * Whether or not the book is a dark manuscript.
+	 */
+	@Getter
+	private final boolean isDarkManuscript;
+
+	/**
+	 * Whether this is the last dark manuscript that was found.
+	 */
+	private boolean isCheckedDarkManuscript;
+
+	/**
+	 * Whether or not a book is present in the player inventory.
+	 * This value is updated whenever the player's inventory changes.
+	 */
+	@Getter
+	@Setter
+	private boolean isInInventory;
+
+	// Static variables initialization
+	static
+	{
+		BY_ID = buildById();
+		BY_NAME = buildByName();
+	}
+
+	/**
+	 * Static method, create a HashMap of books using their id as key.
```

Comment:
Don't say `Static method` in the javadoc, its already there in the method declaration, don't duplicate it. This could just be 
```
/**
 * @return Map of books by id
 */
```
but at that point your just repeating the method declaration.